packages/postgres-database/src/simcore_postgres_database/models/licensed_items.py

```diff
@@ -58,6 +58,11 @@ class LicensedResourceType(str, enum.Enum):
         nullable=False,
         doc="Product name",
     ),
+    sa.Column(
+        "license_key",
+        sa.String,
+        nullable=True,
```

Comment:
`doc` what is a license key? 

TIP:
- Please do not just write "license key" ;-) . 
- What is a license key? what is is used for? who/which part of the code needs it? why is necessary?
- Add an example.
- drop answers and let chatty reformulate the doc nicesly